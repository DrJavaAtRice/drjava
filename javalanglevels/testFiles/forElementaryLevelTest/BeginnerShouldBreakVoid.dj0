/*!DrJava Beginner Level*/

class BeginnerShouldBreakVoid extends Object{
   void badMethod() {
  }
}