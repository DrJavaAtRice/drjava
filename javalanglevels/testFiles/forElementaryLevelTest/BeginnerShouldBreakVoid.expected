// Language Level Converter line number map: dj*->java. Entries: 6
//     1->3         2->4         3->5         4->6         5->7         6->8    
/*!DrJava Beginner Level*/

class BeginnerShouldBreakVoid extends Object{
   public void badMethod() {
  }

  /** This method is automatically generated by the Language Level Converter. */
  public BeginnerShouldBreakVoid() {
    super();
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      BeginnerShouldBreakVoid cast = ((BeginnerShouldBreakVoid) o);
      return true;
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode();
  }
}
